components/SSI/DIDDocument/Keys/index.tsx

```diff
@@ -32,8 +47,8 @@ function Component() {
                 <h3 className={styles.blockHead}>{res[0]}</h3>
                 {res[1].map((element: any) => {
                   return (
-                    //@todo-1 copy to clipboard
-                    <p key={element} className={styles.didkey}>
+                    //@todo-1 copy to clipboard: done
```

Comment:
@DevPH-Wistkey for future reference, when the to-do gets done just delete its comment line - thanks!